jupyter-extension/jupyterlab_pachyderm/pps_client.py

```diff
@@ -68,6 +70,12 @@ def from_notebook(cls, notebook_path: Union[str, Path]) -> "PpsConfig":
 
         requirements = config.get("requirements")
 
+        external_files = []
+        external_files_str = config.get("external_files")
+        if external_files_str:
+            for external_file in external_files_str.strip().split(','):
+                external_files.append(os.path.join(notebook_directory, external_file.strip()))
```

Comment:
Hmm good point. The intention is these external files have no paths and are just names of files in the same directory of the notebook. 

Perhaps throwing an error if the user specifies a path for an external file is appropriate? Changing the path of the external file on upload seems like it could be confusing for users we probably want to avoid that.

Reply:
I adjusted the code to use `Path.name()` with `external_file` thus stripping any path included by the user on accident. This means they will end up getting the same error as the file not existing `external file file.py could not be found in the directory of the Jupyter notebook` which seems descriptive enough for them to fix the problem.